doc/rst/usingchapel/building.rst

```diff
@@ -216,4 +216,19 @@ WARNINGS=0``).
   ========  ================================================================
 
 
+------------------------
+Controlling Build Output
+------------------------
+
+Aside from the default build output, the Chapel compiler makefile also supports
```

Comment:
Capitalize as Makefile?

Reply:
I was torn because the use above (line 202) is lowercase

Reply:
but I think that needs to be updated too because other locations are uppercase